### DISCUSSION TITLE
More difficult bugs to deal with

### DISCUSSION THREAD

**Discussion by mcuee** [avrdudes/avrdude#1170] [General]
Just want to see if there are some good ways to move further with regard to resolving the bugs identified, espcially important ones.

Among the confirmed bugs, some are more difficult and some have higher impact. It will be good to identify them and see if we can find ways to move towards resolving the issues.
https://github.com/avrdudes/avrdude/issues?q=is%3Aopen+is%3Aissue+label%3Abug

**Comment by mcuee** [comment]
It seems to me No 1 may be #425 and it was even listed as for Milestone 7.1 by @dl8dtl. 
* https://github.com/avrdudes/avrdude/issues/425

The title only mentions AVR Dragon, but it seems to affect all modern Microchip tools based on `stk500v2.c`, `jtagmkII.c` and `jtag3.c` if I am correct.

**Comment by mcuee** [comment]
This one is a strange one with a "working fix" which does not seem to be the right fix. I think the impact is not that significant though.
* https://github.com/avrdudes/avrdude/issues/455

**Reply by mcuee** [reply]
This one has been fixed by @stefanrueger with PR #1265.

**Comment by mcuee** [comment]
Another bug which is worth some efforts to sort out is probably the following. I think @stefanrueger has already some good thoughts to sort it out.
* https://github.com/avrdudes/avrdude/issues/1050

**Reply by stefanrueger** [reply]
Will deal with #1050. I am currently in the last thoes to finish the urclock programmer. After that I plan to deal with #1050 and #1152. Shan't be long.
